src/SprykerEco/Shared/CrefoPay/CrefoPayConfig.php

```diff
@@ -247,4 +247,14 @@ public function getCrefoPayPaymentMethodCreditCard3D(): string
     {
         return static::CREFO_PAY_PAYMENT_METHOD_CREDIT_CARD_3D;
     }
+
+    /**
+     * @api
+     *
+     * @return bool
+     */
+    public function getUseIndependentOrderIdForTransaction(): bool
+    {
+        return $this->get(CrefoPayConstants::USE_INDEPENDENT_ORDER_ID_FOR_TRANSACTION, false);
```

Comment:
You can move it to `src/SprykerEco/Service/CrefoPay/CrefoPayConfig.php` directly